README.md

```diff
@@ -1,25 +1,38 @@
+[![SublimeHQ Discord](https://img.shields.io/discord/280102180189634562?label=SublimeHQ%20Discord&logo=discord&style=flat-square)](https://discord.gg/D43Pecu)
```

Comment:
I'd rather link to the forum Resources post than directly to our Discord server.